its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java

```diff
@@ -1,19 +1,18 @@
 /*
- * SonarSource :: HTML :: ITs :: Plugin
- * Copyright (c) 2011-2024 SonarSource SA and Matthijs Galesloot
```

Comment:
Is it ok to remove the guy's name from the license if he was there before?

Reply:
We had a long discussion by email and slack with Nico Peru about it. We got his approval and this PR is the execution.